src/Engine/Graphics/DecalBuilder.cpp

```diff
@@ -267,74 +267,71 @@ bool DecalBuilder::ApplyBloodSplatToTerrain(struct Polygon* terrpoly, Vec3f* ter
     unsigned int NumBloodsplats = bloodsplat_container->uNumBloodsplats;
 
     if (NumBloodsplats > 0) {
-        // loop over blood to lay
-        for (uint i = 0; i < NumBloodsplats; ++i) {
-            // get triangle geometery
-            if (uStripType == 4) {
-                WorldXPosR = triverts[0].vWorldPosition.x;  // left
-                WorldXPosL = triverts[3].vWorldPosition.x;  // right
-                WorldYPosU = triverts[1].vWorldPosition.y;  // bott
-                WorldYPosD = triverts[0].vWorldPosition.y;  // top
-            } else if (uStripType == 3) {
-                if (tri_orient) {
-                    WorldXPosR = triverts->vWorldPosition.x;
-                    WorldXPosL = triverts[2].vWorldPosition.x;
-                    WorldYPosU = triverts[1].vWorldPosition.y;
-                    WorldYPosD = triverts[2].vWorldPosition.y;
-                } else {
-                    WorldXPosR = triverts[1].vWorldPosition.x;
-                    WorldXPosL = triverts[2].vWorldPosition.x;
-                    WorldYPosU = triverts[1].vWorldPosition.y;
-                    WorldYPosD = triverts->vWorldPosition.y;
-                }
+        // get triangle geometery
+        if (uStripType == 4) {
```

Comment:
sorry do we have any idea what these strip types stand for? if we do, can we do an enum?